src/proto/grpc/testing/test.proto

```diff
@@ -47,6 +47,11 @@ service TestService {
   // One request followed by one response.
   rpc UnaryCall(SimpleRequest) returns (SimpleResponse);
 
+  // One request followed by one response. Response has cache control
+  // headers set such that a caching HTTP proxy (such as GFE) can
+  // satisfy subsequent requests.
+  rpc CacheableUnaryCall(SimpleRequest) returns (SimpleResponse);
```

Comment:
Since this isn't defining itself as being cachable, I guess you should add a section (marked as temporary) to the client spec saying to enable caching for the request?